### PR TITLE
UI changes in DressingRoom2 in sync with iOS

### PR DESCRIPTION
**Pull request by aanchal07**
Changed the UI of Dressing Room 2, from which user can purchase items similar to iOS.

**Portrait:**

dressingroom2ui1

dressingroom2ui2

**Landscape:**

dressingroom2ui4

dressingroom2ui3

### CODE REVIEW

**Comment by aanchal07** [comment]
@chhavip : Please review.

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 1]
```diff
@@ -8,7 +8,11 @@
 import android.widget.Button;
 import android.widget.ImageButton;
 import android.widget.ImageView;
+import android.widget.LinearLayout;
 import android.widget.TextView;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 2]
```diff
@@ -8,7 +8,11 @@
...
+import com.akexorcist.roundcornerprogressbar.IconRoundCornerProgressBar;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: same

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 3]
```diff
@@ -27,41 +35,159 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        }
+
+        String faceImageName = getResources().getString(R.string.face);
+        faceImageName = faceImageName + getmDbHandler().getAvatarFace();
+        try {
+            photoNameField = ourRID.getClass().getField(faceImageName);
+            faceView.setImageResource(photoNameField.getInt(ourRID));
+        } catch (NoSuchFieldException | IllegalAccessException
+                | IllegalArgumentException e) {
+            // TODO Auto-generated catch block
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: same here and below

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 4]
```diff
@@ -27,41 +35,159 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        powerbarTelepathy.setIconImageResource(R.drawable.icon_telepathy);
+        powerbarTelepathy.setProgress(mDbHandler.getTelepathy());
+
+
         final String value = getIntent().getExtras().getString(getResources().getString(R.string.feature));
-        TextView tv = (TextView) findViewById(R.id.textViewSelectFeature);
-        tv.setText(R.string.select_feature_title + value);
-        ImageButton left = (ImageButton) findViewById(R.id.leftSelectFeature);
-        ImageButton right = (ImageButton) findViewById(R.id.rightSelectFeature);
+//        TextView tv = (TextView) findViewById(R.id.textViewSelectFeature);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove if not needed

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 5]
```diff
@@ -27,41 +35,159 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        powerbarInvisibility.setProgress(mDbHandler.getInvisibility());
+
+        IconRoundCornerProgressBar powerbarStrength = (IconRoundCornerProgressBar) findViewById(R.id.powerbarStrength);
+        powerbarStrength.setIconImageResource(R.drawable.icon_strength);
+        powerbarStrength.setProgress(mDbHandler.getStrength());
+
+        IconRoundCornerProgressBar powerbarTelepathy = (IconRoundCornerProgressBar) findViewById(R.id.powerbarTelepathy);
+        powerbarTelepathy.setIconImageResource(R.drawable.icon_telepathy);
+        powerbarTelepathy.setProgress(mDbHandler.getTelepathy());
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove extra line

**Comment by aanchal07** [comment]
@chhavip : Made the changes as you pointed out.

**Comment by chhavip** [comment]
@aanchal07 Please resolve the conflicts before the changes can be merged.

**Comment by aanchal07** [comment]
@chhavip : I am getting merge conflicts after squashing the commits. I have tried 2-3 times. I cannot reduce to less than 2 commits.

**Pull request merged**